src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -1606,9 +1606,13 @@ private void InheritNullableStateOfFieldOrProperty(int targetContainerSlot, int
             }
             else if (EmptyStructTypeCache.IsTrackableStructType(fieldOrPropertyType.TypeSymbol))
             {
-                InheritNullableStateOfTrackableStruct(fieldOrPropertyType.TypeSymbol, 
-                                                      GetOrCreateSlot(fieldOrProperty, targetContainerSlot),
-                                                      valueContainerSlot > 0 ? GetOrCreateSlot(fieldOrProperty, valueContainerSlot) : -1, isByRefTarget);
+                var slot = GetOrCreateSlot(fieldOrProperty, targetContainerSlot);
```

Comment:
Did you want to add an assert or comment for this? Eventually I'd prefer we use an enum or something similar to make these cases clear, but until then it at least seems useful to have an asset or comment.